custom_functions/mark_evidence.py

```diff
@@ -0,0 +1,66 @@
+def mark_evidence(container=None, object_id=None, content_type=None, **kwargs):
+    """
+    Mark an object as Evidence in a container
+    
+    Args:
+        container (CEF type: phantom container id): Container ID or Container Object
+        object_id (CEF type: *): Id of object to be added - artifact id, note id, etc.
+        content_type (CEF type: *): The content type of the object to add as evidence which must be one of the following:
+                        
+                        containerattachment
```

Comment:
Sorry I missed this the first time, but the containerattachment type seems to be difficult to use. I think we should add a new type called "vault_id" which should accept the sha1 hash of the file. That is the standard way to pass around references to a vault file. Then we should use code like the following to get the id:

```python
    vault_id = '8fe50491d77eace578cfd2862a14c93dc902be43'
    success, message, info = phantom.vault_info(
        vault_id=vault_id
    )

    phantom.debug(
        'phantom.vault_info results: success: {}, message: {}, info: {}'\
        .format(success, message, info)
    )
    
    container_attachment_id = info[0]['id']
    phantom.debug(container_attachment_id)

Reply:
Had to make input content_type = vault_id and then translate in the code since VPE was having a problem with "containerattachment"